docs/architecture/microservices/net-core-net-framework-containers/net-framework-container-scenarios.md

```diff
@@ -15,31 +15,29 @@ In most cases for this scenario, you will not need to migrate your existing appl
 
-In addition, the [Windows Compatibility Pack](../../../core/porting/windows-compat-pack.md) was released on NOV-2017 to extend the API surface available for .NET Standard 2.0 on Windows. This pack allows recompiling most existing code to .NET Standard 2.x with little or no modification, to run on Windows.
+In addition, the [Windows Compatibility Pack](../../../core/porting/windows-compat-pack.md) extends the API surface available for .NET Standard 2.0 on Windows. This pack allows recompiling most existing code to .NET Standard 2.x with little or no modification, to run on Windows.
 
 However, even with that exceptional progression since .NET Standard 2.0 and .NET Core 2.1, there might be cases where certain NuGet packages need Windows to run and might not support .NET Core. If those packages are critical for your application, then you will need to use .NET Framework on Windows Containers.
 
 ## Using .NET technologies not available for .NET Core
 
-Some .NET Framework technologies are not available in the current version of .NET Core (version 2.2 as of this writing). Some of them will be available in later .NET Core releases (.NET Core 2.x), but others do not apply to the new application patterns targeted by .NET Core and might never be available.
+Some .NET Framework technologies are not available in the current version of .NET Core (version 3.1 as of this writing). Some of them might be available in later .NET 5 releases, but others don't fit the new application patterns targeted by .NET Core and might never be available.
 
-The following list shows most of the technologies that are not available in .NET Core 2.x:
+The following list shows most of the technologies that are not available in .NET Core 3.1:
 
 - ASP.NET Web Forms. This technology is only available on .NET Framework. Currently there are no plans to bring ASP.NET Web Forms to .NET Core.
 
-- WCF services. Even when a [WCF-Client library](https://github.com/dotnet/wcf) is available to consume WCF services from .NET Core, as of mid-2017, the WCF server implementation is only available on .NET Framework. This scenario might be considered for future releases of .NET Core, there are even some APIs considered for inclusion in the [Windows Compatibility Pack](../../../core/porting/windows-compat-pack.md).
+- WCF services. Even when a [WCF-Client library](https://github.com/dotnet/wcf) is available to consume WCF services from .NET Core, as of Feb-2020, the WCF server implementation is only available on .NET Framework. This scenario might be considered for future releases of .NET Core, there are even some APIs considered for inclusion in the [Windows Compatibility Pack](../../../core/porting/windows-compat-pack.md).
```

Comment:
Sure not now, I prefer to leave it like that, b/c it might change and it's easier to spot, with a date attached, as something that has to be reviewed.